packages/wallet-utils-contract/test/unit/walletContract.spec.js

```
@@ -0,0 +1,171 @@
+
+      beforeEach(() => {
+        rawTxMetadataDocument = {
+          keyIndex: 0,
+          encryptionKeyIndex: 100,
+          encryptedMetadata: crypto.randomBytes(64),
+        };
+      });
+
+      describe('keyIndex', () => {
+        it('should be defined', async () => {
+          delete rawTxMetadataDocument.keyIndex;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)